buildSrc/build.gradle.kts

```diff
@@ -8,4 +8,8 @@ repositories {
 
 dependencies {
     implementation("com.google.guava:guava:28.0-jre")
-}
\ No newline at end of file
+}
+
+kotlinDslPluginOptions {
```

Comment:
So this means, we're using experimental features, so it's going to ignore giving a warning that's occurring?

Reply:
This only applies to the use of the `kotlin-dsl` plugin within the various gradle helper functions in `buildSrc`. The `kotlin-dsl` plugin is still considered experimental. We've always been using it. I just finally got annoyed with seeing the warning printed by gradle every time it ran.